cluster-provision/images/vm-image-builder/OWNERS

```diff
@@ -0,0 +1,6 @@
+filters:
+  ".*":
+    reviewers:
+      - ormergi
```

Comment:
i think we should use something like network-reviewers
(on both lines, as other OWNER files are doing please)

Reply:
This adds ormergi to the reviewers and approvers of this specific folder. I do not see why it is worth creating a special group for this at the moment, especially when only one is included in it.

Reply:
no problem, just saying what i saw on other files in some kubevirt repos, and on other habits from the past.